src/ui/layout/index.md

```diff
@@ -188,6 +188,71 @@ class MyApp extends StatelessWidget {
   or you can build your own set of custom widgets.
 {{site.alert.end}}
 
+#### Cupertino apps
+
+To create a `Cupertino` app, use a [`CupertinoPageScaffold`][] widget.
+
+Unlike `Material`, it doesn't provide a default banner or background color.
+You need to set these yourself.
+
+* To set default colors, you add a [`CupertinoThemeData`][].
```

Comment:
As a reader, it's not super clear where I'm adding this theme data object, or that I need to do anything with it.

```suggestion
* To set default colors, pass in a configured [`CupertinoThemeData`][]
  to your app's `theme` property.
```